features/rni.feature

```diff
@@ -0,0 +1,202 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
```

Comment:
Deze Given moet nog geïmplementeerd worden.

Reply:
Toch krijg ik hier de volgende melding op als ik het 1e scenario run:

![image](https://user-images.githubusercontent.com/20185784/190105329-3409dde3-5c71-42d9-ba1e-de5b3857fd9c.png)

Reply:
Wat hier fout gaat is dat de regular expression die wordt gebruikt om de naam van de tabel te parsen te strak is gedefinieerd. Volgens de regular expression mag de naam geen `-` teken hebben. Dit wordt toegevoegd.